tools/bug-report/main.go

```diff
@@ -0,0 +1,29 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"istio.io/istio/tools/bug-report/pkg/bugreport"
+)
+
+func main() {
```

Comment:
Would prefer this as part of istioctl - users won't want to download another binary

Reply:
i had structured it for that and was going to add it in a followup, but may as well do it now.